libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/theme/components/IconButton.kt

```diff
@@ -38,13 +39,13 @@ fun IconButton(
     onClick: () -> Unit,
     modifier: Modifier = Modifier,
     enabled: Boolean = true,
+    colors: IconButtonColors = IconButtonDefaults.iconButtonColors(
```

Comment:
Technically you could have used a `LocalCompositionProvider` and set the `LocalContentColor` to `iconSecondary` too. But I think it makes sense to be able to customize these here too.